### ISSUE TITLE
cannot get this working!

### ISSUE THREAD

**Issue by comdraft-de** [mischnic/parcel-namer-preserve-structure#5]
Tested with different configs in parcel serve and parcel build, nothing works.
No errors or warns, it simply does nothing.

**Comment by mischnic** [comment]
Are you using Parcel 2 and a parcelrc as described in the Readme?

**Comment by comdraft-de** [comment]
Yes, all as described.

**Comment by comdraft-de** [comment]
Ok, I think I have some weird old config and must clean up my package.json. After this I'll try it again.

**Comment by comdraft-de** [comment]
I made a cleanup and try it again, with different cli flags and also without any flag. But the result is not as aspected. You can see it in the screenshots.
![parcel_src_folder](https://user-images.githubusercontent.com/45310335/103342726-98892880-4a8a-11eb-97ac-ffb88329b739.png)
![parcel_dist_folder](https://user-images.githubusercontent.com/45310335/103342759-a474ea80-4a8a-11eb-8a9d-90fe695ae159.png)

But in this moment I'm not shure if Parcel is the right build tool for me. It causes to much problems when processing sass files. In Parcel V1 it works only in serve mode and breaks in build mode. This was the reason, why I upgrade to V2. Now the sass process is working in both modes but it is extreme slow.
This is sad, because Parcel seems to be simple to use and lern.

**Comment by mischnic** [comment]
Please provide a small complete code example of your situation.

If you've found a case where Sass seems to be slower than it should be, open an issue here https://github.com/parcel-bundler/parcel/issues with an example

**Comment by comdraft-de** [comment]
Hi Niklas,
sorry for late reply and a happy new year for you!

A 'small complete code example' is a paradox, I'm working on a WordPress theme. ;) But for now all seems fine!

I made a fresh install of my node modules and the most is working fine now and as expected. I also changed the way to compile the scss files. The first time I've imported the site.scss inside the site.js, I changed this by adding the site.scss to the parcel serve script instead. This was working fine in serve and also in build mode, I also get the entire folder structure from src to dist for js and css files. To get also the fonts folder I reinstall your plugin and this also seems to work fine for the first moment. The folder was generated in dist and all font-files inside. But I had also a flag for the public url which causes that the font path within the generated site.css was incorrect. So I removed this flag also and all is working fine and fast for now.

Thank's for your support!
